start-deployPaper

```diff
@@ -3,9 +3,10 @@
 . /start-utils
 
 export SERVER=paper_server-${VANILLA_VERSION}.jar
+PAPERBUILD=${PAPERBUILD:-LATEST}
```

Comment:
If you include `PAPERBUILD` in the filename then people can upgrade by bumping the variable. I'm thinking this would do it:
```suggestion
PAPERBUILD=${PAPERBUILD:-LATEST}
export SERVER=paper_server-${VANILLA_VERSION}-${PAPERBUILD}.jar
```

Reply:
Its an great idea